library/init/meta/interactive.lean

```diff
@@ -66,6 +81,7 @@ private meta def parser_desc_aux : expr → tactic (list format)
 | `(ident)  := return ["id"]
 | `(ident_) := return ["id"]
 | `(qexpr) := return ["expr"]
+| `(location) := return ["loc?"]
```

Comment:
Oh I see, it handles `return` but not `pure`. I pushed a fix, it should work now out of the box.